pylatex/utils.py

```diff
@@ -148,18 +137,17 @@ def dumps_list(l, *, escape=True, token='%\n', mapper=None, as_content=True):
         Whether to escape special LaTeX characters in converted text.
     token : str
         The token (default is a newline) to separate objects in the list.
-    mapper: callable
-        A function that should be called on all entries of the list after
-        converting them to a string, for instance bold
+    mapper: callable or `list`
+        A function or a list of functions that should be called on all
+        entries of the list after converting them to a string, for instance
+        bold
```

Comment:
Could you add backticks around bold? I know it wasn't there before but I just noticed this now. If it gives an error try 
```
`~.bold`
```